CONTRIBUTING.md

```diff
@@ -1,3 +1,14 @@
 # Contributing to Flyte
 
-For information related to contributing to Flyte, please check out the [Contributing to Flyte](https://docs.flyte.org/en/latest/community/contribute.html) section of the documentation.
+For information related to contributing to Flyte, please check out the [Contributing to Flyte](https://docs.flyte.org/en/latest/community/contribute/index.html) section of the documentation.
+
+## Recommendation Order (For Beginners)
+* Setup dev environment 
+* Read the following and try to run 10 examples
```

Comment:
Do you think we could lower the barrier here? Any number is arbitrary at this point so maybe something like "Run at least one example and pay close attention to effects: generated outputs, the Graph view, tasks logs, etc. Repeat with as many examples as you need to have an initial understanding of what an execution looks like" wdyt?